packages/fluentui/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -156,10 +156,9 @@ const Checkbox: React.FC<WithAsProp<CheckboxProps>> &
       {Icon.create(icon, {
```

Comment:
Box?

Reply:
Not sure if Box is valid here, as from client's perspective I think icon is the valid slot

Reply:
I don't see any reasons to be honest 🤔  You will not use `book` for as checkmark